### ISSUE TITLE
How to help/contribute?

### ISSUE THREAD

**Issue by Kranzes** [microg/GmsCore#1680]
How can I contribute to this project with little to non experience with java/kotlin?

**Comment by MagicRB** [comment]
Hi @Kranzes small world. I think that going through the issues and trying to clean them up would be a good start. There are quite a few issues with not enough information. Think X doesn't work, heeeelp! Trying to contact the authors and asking to follow up would imo help the actuall microg maintainers a lot. It's housekeeping that no one wants to do but must be done. I'll start myself once I have more free time.

**Comment by ale5000-git** [comment]
Also the most important thing is to have people doing reports with the latest version of microG.
Reports with old versions are mostly useless.

**Comment by MagicRB** [comment]
Yeah, cleaning up issues, removing old reports that aren't useful. Though it would be nice if someone could actually close them, I found a few that people commented `can be closed now` but they haven't yet been closed.

**Comment by MagicRB** [comment]
@mar-v-in could someone get issue mod perms? So that we can clean this up?

**Comment by ale5000-git** [comment]
I can close ticket directly, but I may have missed something.
Could you please link them here?

**Comment by MagicRB** [comment]
Ill try to find it

**Comment by ILoveOpenSourceApplications** [comment]
@ale5000-git, here are some issues you can close including this one itself. 
#2419 
#2361
#2360 
#2339 
#2334 
#2330 
#2328 
#2327 
#2286 
#2145 
#2140
#2095
#2089 
#2085 
#2078 
#2075 
#2054 
#1931 
#1928 
#1919 
#1914 
#1911 
#1905 
#1904 
#1844 
#1841 
#1810 
#1769 
#1758 
#1732 
#1680
#1676 
#1581 
#1580 
#1560 
#1557 
#1514 
#1510 
#1394 
#1147 
#1110 
#1106 
#1060 
#1009 
#936 
#921 
#885 
#748 
#708 
#697 
#574 
I lost my attention span after this. So many issues are unanswered and I can't confirm whether those are resolved or not. Anyways,  in the above mentioned issues, most of em can be closed but if possible keep an eye, just in case, if I've mentioned anything that is still relevant. 

This issues repo require a thorough clean up and a lot of patience and time from someone.

**Comment by Axolord** [comment]
Maybe a bot that marks issues as stale if no one comments for a while and closes them after another grace period would be useful..?

That would at least clean up the massive amount of really old, opened issues.

**Comment by mar-v-in** [comment]
The idea is to no longer have individual app reports in GitHub issues, but rather do them somewhere else, similar to wine appdb. And then GitHub issues would be limited to technical issue with microG, independent of Apps. This would allow or help with cleaning up drastically.

I'm not a big fan of automatic closing of issues after time, but I agree it might be helpful here as it's unclear for many issues if they're still valid.
